test/WebSites/BasicWebSite/Areas/Area1/Views/RemoteAttribute_Home/Details.cshtml

```diff
@@ -64,13 +61,5 @@
 </div>
 
 <p>
-    @Html.ActionLink("Create a new one", "Create") |
-    @if (string.Equals("Aria", areaName, StringComparison.OrdinalIgnoreCase))
-    {
-        @Html.RouteLink("Go back to home", "Home")
-    }
-    else
-    {
-        @Html.RouteLink("Go to Aria home", "AriaHome")
-    }
+    @Html.ActionLink("Create a new one", "Create") | @Html.RouteLink("Go to Area1 home", "AriaHome")
```

Comment:
This view is in its own area `Area1` as opposed to being in the root shared folder in which case this piece of code made sense but here its not required. Also this is not a pivot in this case right?